Makefile

```diff
@@ -78,6 +78,16 @@ ifneq ("$(FIPS)","")
 FIPS_TAG := fips
 FIPS_MESSAGE := with-FIPS-support
 RELEASE = teleport-$(GITTAG)-$(OS)-$(ARCH)-fips-bin
+GOEXPERIMENT = boringcrypto
+OPENSSL_FIPS = 1
+export GOEXPERIMENT OPENSSL_FIPS
+ifeq ($(BUILDBOX_MODE),cross)
+# We need to set CGO_ENABLED=0 when building rdpclient as the build of
+# boring-sys builds and runs a Go program which fails for odd reasons if
+# CGO_ENABLED is on. It also needs to know the cross-compiler sysroot
+# to properly cross-compile.
+RDPCLIENT_ENV = CGO_ENABLED=0 BORING_BSSL_FIPS_SYSROOT=$(CROSSTOOLNG_SYSROOT)
```

Comment:
I don't understand how CGO_ENABLED does anything here. The cargo tool doesn't know anything about Go or Cgo

Reply:
I knew this would be confusing, and I did try to describe it in the comment above. I don't want to explain too much because I would like to see how much it takes for you to arrive at the conclusion. What if the comment above said:

```
# rdpclient uses the boring-sys crate which builds boringssl. The build of boringssl builds and runs a
# Go program which fails to build if CGO_ENABLED=1 when cross-compiling.
# The boring-sys crate also requires BORING_BSSL_FIPS_SYSROOT point to the sysroot when
# cross-compiling.
```

?

Reply:
Are you saying that Go is required in order to build any program that uses boringssl? That would be news to me.

Reply:
https://github.com/google/boringssl/blob/853ca1ea1168dff08011e5d42d94609cc0ca2e27/crypto/fipsmodule/CMakeLists.txt#L206

Reply:
TIL! (I might link to https://github.com/google/boringssl/blob/master/crypto/fipsmodule/FIPS.md#integrity-testing so this is clear for the next reader)